src/ui/service/notifications/registry/handler.go

```diff
@@ -159,7 +159,7 @@ func filterEvents(notification *models.Notification) ([]*models.Event, error) {
 		}
 
 		// pull and push manifest by docker-client or vic
-		if (strings.HasPrefix(event.Request.UserAgent, "docker") || strings.HasPrefix(event.Request.UserAgent, vicPrefix)) &&
+		if (strings.HasPrefix(event.Request.UserAgent, "docker") || strings.HasPrefix(event.Request.UserAgent, vicPrefix) || strings.HasPrefix(event.Request.UserAgent, "jib")) &&
```

Comment:
Prefix checking and action checking can be abstracted to a separate handler/function?

e.g:

  `EventHandler.AcceptAgents(...string).AcceptActions(...string).Handle(event)?`

This is just a suggestion, you can try your own one.